packages/cozy-mespapiers-lib/src/components/ModelSteps/widgets/InputTextAdapter.jsx

```diff
@@ -1,8 +1,10 @@
 import React, { useState, useEffect, useMemo } from 'react'
 import PropTypes from 'prop-types'
 import InputMask from 'react-input-mask'
+import { InputAdornment } from '@material-ui/core'
```

Comment:
You should not import from `mui-core` directly. Cozy-UI should export it.

Reply:
https://github.com/cozy/cozy-ui/pull/2291